uat/custom-components/client-mosquitto-c/src/GRPCControlServer.cpp

```diff
@@ -260,6 +260,13 @@ Status GRPCControlServer::PublishMqtt(ServerContext *, const MqttPublishRequest
         p_payload_format_indicator = &payload_format_indicator;
     }
 
+    int message_expire_interval;
```

Comment:
It looks like my changes. Need rebase your branch on top of GGMQ-182-set-unset-message-expiry-interval or cherry-pick your commits to second feature branch.

Reply:
fixed by rebase